Makefile

```diff
@@ -10,6 +10,12 @@ REGISTRY_NAMESPACE ?=
 build: $(SOURCES) ## Build binary from source
 	go build -i -ldflags="-s -w" -o _out/hyperconverged-cluster-operator ./cmd/manager
 
+install: build ## Install binaries into the GOBIN folder to make it easier discoverable for openshift ci image streams
+	cp _out/hyperconverged-cluster-operator ${GOBIN}
+
+okd-e2e: ## Test target for e2e tests against latest openshift releases
+	echo "nothing yet"
```

Comment:
Add tests to `hack/functest.sh` so you can call the `functest` recipe.  I want to clear out that file in this [PR](https://github.com/kubevirt/hyperconverged-cluster-operator/pull/180/files#diff-d62ab515dc060559d87410b1ff672465L3).